bsdate.go

```diff
@@ -217,3 +221,59 @@ func (d date) isValid() bool {
 	}
 	return true
 }
+
+func (d date) getGregorianDate() time.Time  {
+	var daysAfterJanFirstOfGregorianYear = 0 //we will add all the days that went by since the 1st.
+	                                         //January and then we can get the gregorian Date
+	var gregorianYear int
+	var nepaliMonthToCheck  = d.Month
+	var nepaliYearToCheck  = d.Year
+
+
+	//get the correct year
+	//after the month of Poush (9) or in Poush but after 1st Jan in that BS year we have to substract 56 years, elst 57
```

Comment:
```suggestion
	//after the month of Poush (9) or in Poush but after 1st Jan in that BS year we have to substract 56 years, else 57
```

Reply:
`Poush` or `Paush` - spell it consistently however you spelt it in the real code.